netpyne_ui/netpyne_geppetto.py

```diff
@@ -303,27 +303,27 @@ def create_docker_container(self, args):
         with redirect_stdout(sys.__stdout__):
             try:
                 # delete previous data
-                subprocess.call(['rm', '-r', './docker'])
-                subprocess.call(['mkdir', './docker'])
-                subprocess.call(['mkdir', './docker/mod'])
+                subprocess.call(['rm', '-r', './GUIdocker'])
```

Comment:
@rodriguez-facundo We talked about using something like this https://docs.python.org/3/library/tempfile.html ?  What happened in the end?

Reply:
Ok, this is why I asked you if this folder would store the final outputs or the progress. You said it would be just the scripts and, therefore, I suggested to use that python temp API. However, if this is something you want to keep we should discuss a few things.:
- Are you explaining this to the user somehow? Explaining to him that this feature will create a docker image and also, the scripts that generated the docker image, where these scripts and resources can be found, what he can do with them, etc. 
- If you want to keep these files wouldn't it better to name the folder sth like Docker_timestamp so he can have several folders?
- Is this actually useful for a regular user or for yourself? :) What would a regular user do with these scripts? If this is useful for a regular user, why don't you ask for the folder path as we do for the rest of the paths? This way he can also choose if he wants to keep the scripts or just the image.

Reply:
yeah, sure. I did this just because of a guy send me an email telling me that a model I had in github that I created using netpyne was not working for him. So we had to iterate over the model and the neuron installation and the netpyne version, until we finally got the simulation running. So, I thought, "what if I just give to this guy a docker image so he and I don't waste time trying to figure out his neuron version, his PYTHON_PATH, the netpyne version, etc..." So I created this. It was basically for myself...
Y guess I could add a question icon with a complete explanation of what it does and how it works.

Reply:
and a field to specify the folder path

Reply:
As we have the sprint meeting tomorrow, let's discuss with the guys the best approach for this tomorrow